CollapseRegionExtension/RegionCommands.cs

```diff
@@ -164,6 +164,14 @@ private async Task<(IOutliningManager manager, IEnumerable<ICollapsible> regions
                 {
                     regions.Add(region);
                 }
+                if (text.StartsWith("<!--"))
+                {
+                    var textLow = text.ToLower();
+                    if (textLow.Contains("region"))
+                    {
+                        regions.Add(region);
+                    }
+                }

```

Comment:
this will also affect xaml multi-line comments that for some reason contain word 'region'. Example : 
`<Application x:Class="WpfHost.App"
             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
             StartupUri="MainWindow.xaml">
    <Application.Resources>
        <!-- random text
        
        region random text 2
                -->
    </Application.Resources>
</Application>
`
Probably not important enough to worry about